github/event_types.go

```diff
@@ -165,6 +165,28 @@ type ProjectColumnChange struct {
 	} `json:"name,omitempty"`
 }
 
+// TeamChange represents the changes when a team has been edited.
+type TeamChange struct {
+	Description *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"description,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"name,omitempty"`
+	Privacy *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"privacy,omitempty"`
+	Repository *struct {
+		Permissions *struct {
+			From *struct {
+				Admin *string `json:"admin,omitempty"`
+				Push  *string `json:"push,omitempty"`
+				Pull  *string `json:"pull,omitempty"`
```

Comment:
> I was going from most permissive to least

I see, that makes sense. Probably even more sense than the order the docs have them in.

We could also re-arrange them to be `Pull`, `Push`, and `Admin` last. That way it's from least permissive to most, and consistent with:

- https://github.com/google/go-github/blob/ebfec748347a9af6793c723f8859afcd906860fb/github/repos_collaborators.go#L86-L88
- https://github.com/google/go-github/blob/ebfec748347a9af6793c723f8859afcd906860fb/github/orgs_teams.go#L263-L265

But I noticed that some sample responses in https://developer.github.com/v3/repos/ also have them in the alphabetical order:

```JSON
"permissions": {
  "admin": false,
  "push": false,
  "pull": true
},
```

So, there's really no single best way, they're all slightly inconsistent. I'll leave it to you to decide what to go with. I just wanted to bring your attention here and make sure the ordering choice was deliberate rather than accidental.